app/Utils/ConfReport.php

```diff
@@ -281,7 +281,7 @@ class ConfReport
 		'user_privileges/' => ['type' => 'IsWritable', 'testCli' => true],
 		'user_privileges/tabdata.php' => ['type' => 'IsWritable', 'testCli' => true],
 		'user_privileges/menu_0.php' => ['type' => 'IsWritable', 'testCli' => true],
-		'user_privileges/user_privileges_1.php' => ['type' => 'IsWritable', 'testCli' => true, 'required'=>false],
+		'user_privileges/user_privileges_1.php' => ['type' => 'IsWritable', 'testCli' => true, 'mode'=> 4],
```

Comment:
```suggestion
		'user_privileges/user_privileges_1.php' => ['type' => 'IsWritable', 'testCli' => true, 'mode'=> self::MODE_{description}],
```
use constants in the file